### PR TITLE
Add reference to LICENSE in README

### PR DESCRIPTION
**Pull request by legionth**
Due the current state of the [LICENSE.md](./docs/LICENSE.md), the license can't be [shown properly on GitHub](https://help.github.com/articles/licensing-a-repository/).

We should at least state in the README where to find the license file. This PR will take care of that.

### CODE REVIEW

**Comment by Amstutz** [comment]
> Due the current state of the LICENSE.md, the license can't be shown properly on GitHub. 

What do you mean by that? I have now trouble looking up the license in github, but I guess this is not the point, right? Is something off with the content of the license?

> We should at least state in the README where to find the license file. This PR will take care of that.

That would certainly not hurt.

**Review by Amstutz — APPROVED** [review]

**Comment by legionth** [comment]
Hey @Amstutz,

Checkout the referenced link above: https://help.github.com/articles/licensing-a-repository/

GitHub uses a similar approach like: https://github.com/benbalter/licensee
Which will search for well known licenses like MIT or GPL etc.

Because of the structure of the current Ilias LICENSE the process to identfy the LICENSE fails and won't diplay on the front page e.g.:

license-example

Normally a GPL License will begin like this:

```
                    GNU GENERAL PUBLIC LICENSE
                       Version 3, 29 June 2007
```

But the license in ILIAS **begins with an addtion** to the GPL, which will result in a non-display
of the license on the front page on GitHub.
Additionally ILIAS won't be listed if a user uses the GitHub search by using: `license:gpl`

This PR won't get rid of that, just makes it easier (by a reference in the README) for the user to find the license :smiley: 

Feel free to give me your thoughts on this :+1:

**Comment by Amstutz** [comment]
Thanks for your feedback/summary, and even more, thanks for pointing this out. We will discuss this in the TB. I hope we can get rid of this issue.

**Comment by legionth** [comment]
tl;dr: There is nothing wrong with the LICENSE file, but it could be improved to collaborate better with GitHub in an upcoming step.

**Comment by mjansenDatabay** [comment]
@Amstutz @legionth @chfsx : As far as I remember I already put this topic on the TB agenda some weeks ago.

**Comment by Amstutz** [comment]
@mjansenDatabay : I just checked the "postponed" items on our agenda. Of course, you are right. I guess this would be the moment to properly pro this and discuss it during the next meeting ;-). And, thanks to you too I guess ;-)

**Comment by alex40724** [comment]
+1 (oder verschieben)

**Comment by chfsx** [comment]
Hi @legionth, the @ILIAS-eLearning/technical-board decided to introduce the "GitHub"-Way and just added a `LICENSE`-File in trunk und release_5-3. Therefore we close this request here, thanks for proposing this solution and the hint to the "GitHub"-Way!
